Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml

```diff
@@ -7,5 +7,8 @@
   - type: Projectile
     damage:
       types:
-        Piercing: 49
+        Piercing: 60
         Structural: 30
+    ignoreResistances: true
+  - type: StaminaDamageOnCollide
+    damage: 100
```

Comment:
People are just going to shotgun this and then get a free kill.

Reply:
maybe lower the stamina damage. 70 maybe?

Reply:
shouldn't snipers be finished implementing first before going and making balance changes